src/common/get-configuration-file-model/get-configuration-file-model.global-override-for-injectable.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import assert from "assert";
+import { action } from "mobx";
+import path from "path";
+import { getGlobalOverride } from "../test-utils/get-global-override";
+import getConfigurationFileModelInjectable from "./get-configuration-file-model.injectable";
+import type Config from "conf";
+import readJsonSyncInjectable from "../fs/read-json-sync.injectable";
+import writeJsonSyncInjectable from "../fs/write-json-sync.injectable";
+
+export default getGlobalOverride(getConfigurationFileModelInjectable, (di) => {
```

Comment:
That's one complicated fake as global override. It also hides thrown errors with `try/catch`. Sure about this? :)

Reply:
Yup, this behaves similar to how we use the actual dependency. Sure I could write is slightly differently, but it is correct enough to have the tests behave as expected.